lib/StripeResource.js

```diff
@@ -261,6 +271,9 @@ StripeResource.prototype = {
     // But never sleep less than the base sleep seconds.
     sleepSeconds = Math.max(initialNetworkRetryDelay, sleepSeconds);
 
+    // And never sleep less than the time the API asks us to wait
+    sleepSeconds = Math.max(sleepSeconds, retryAfter || 0);
```

Comment:
Instead of doing `retryAfter || 0` can we just set a default parameter for `retryAfter` in the function declaration?

Reply:
I'd rather not, since default params don't work with `null`. `Math.max` _does_ work with `null`, though it behaves unexpectedly (to me at least) with `NaN` (`Math.max(2, NaN) === NaN`). Using `|| 0` is more robust there, even if it isn't an expected program state.